tests/qos/qos_sai_base.py

```diff
@@ -754,18 +754,24 @@ def __buildTestPorts(self, request, testPortIds, testPortIps, src_port_ids,
         dstVlan3 = dst_test_port_ips[dstPort3]['vlan_id'] if 'vlan_id' in dst_test_port_ips[dstPort3] else None
         srcPort = srcPorts[0] if src_port_ids else src_test_port_ids[srcPorts[0]]
         srcVlan = src_test_port_ips[srcPort]['vlan_id'] if 'vlan_id' in src_test_port_ips[srcPort] else None
+
+        src_port_ip = src_test_port_ips[srcPorts[0] if src_port_ids else src_test_port_ids[srcPorts[0]]]
         return {
          "dst_port_id": dstPort,
          "dst_port_ip": dst_test_port_ips[dstPort]['peer_addr'],
+         "dst_port_ipv6": dst_test_port_ips[dstPort]['peer_addr_ipv6'],
          "dst_port_vlan": dstVlan,
          "dst_port_2_id": dstPort2,
          "dst_port_2_ip": dst_test_port_ips[dstPort2]['peer_addr'],
+         "dst_port_2_ipv6": dst_test_port_ips[dstPort2]['peer_addr_ipv6'],
          "dst_port_2_vlan": dstVlan2,
          'dst_port_3_id': dstPort3,
          "dst_port_3_ip": dst_test_port_ips[dstPort3]['peer_addr'],
+         "dst_port_3_ipv6": dst_test_port_ips[dstPort3]['peer_addr_ipv6'],
```

Comment:
We are seeing this error. What is the expectation of this particular key? The qos-sai-base completely disables the ipv6. So there will be no ipv6 in the DUT at all.

`         "src_port_id": srcPort,
         "src_port_ip": src_port_ip["peer_addr"],
         "src_port_ipv6": src_port_ip["peer_addr_ipv6"],
         "src_port_vlan": srcVlan
        }
E       KeyError: 'peer_addr_ipv6'

dstPort    = 0
dstPort2   = 2
dstPort3   = 3
dstPorts   = [0, 2, 3]
dstVlan    = None
dstVlan2   = None
dstVlan3   = None
dst_dut_port_ids = {0: [0, 1, 2, 3, 4, 5, ...]}
dst_dut_port_ips = {0: {0: {'peer_addr': '192.168.0.2'}, 1: {'peer_addr': '192.168.0.3'}, 2: {'peer_addr': '192.168.0.4'}, 3: {'peer_addr': '192.168.0.5'}, ...}}
dst_port_ids = None
dst_test_port_ids = [0, 1, 2, 3, 4, 5, ...]
dst_test_port_ips = {0: {'peer_addr': '192.168.0.2'}, 1: {'peer_addr': '192.168.0.3'}, 2: {'peer_addr': '192.168.0.4'}, 3: {'peer_addr': '192.168.0.5'}, ...}
get_src_dst_asic_and_duts = {'all_asics': [<SonicAsic 0>], 'all_duts': [<MultiAsicSonicHost cmono-t0-dut>], 'dst_asic': <SonicAsic 0>, 'dst_asic_index': 0, ...}
request    = <SubRequest 'dutConfig' for <Function testQosSaiDscpQueueMapping[single_asic-ipv4]>>
self       = <tests.qos.test_qos_sai.TestQosSai object at 0x7f3fc262ca90>
srcPort    = 1
srcPorts   = [1]
srcVlan    = None
src_dut_port_ids = {0: [0, 1, 2, 3, 4, 5, ...]}
src_dut_port_ips = {0: {0: {'peer_addr': '192.168.0.2'}, 1: {'peer_addr': '192.168.0.3'}, 2: {'peer_addr': '192.168.0.4'}, 3: {'peer_addr': '192.168.0.5'}, ...}}
src_port_ids = None
src_port_ip = {'peer_addr': '192.168.0.3'}
src_test_port_ids = [0, 1, 2, 3, 4, 5, ...]
src_test_port_ips = {0: {'peer_addr': '192.168.0.2'}, 1: {'peer_addr': '192.168.0.3'}, 2: {'peer_addr': '192.168.0.4'}, 3: {'peer_addr': '192.168.0.5'}, ...}
testPortIds = {0: {0: [0, 1, 2, 3, 4, 5, ...]}}
testPortIps = {0: {0: {0: {'peer_addr': '192.168.0.2'}, 1: {'peer_addr': '192.168.0.3'}, 2: {'peer_addr': '192.168.0.4'}, 3: {'peer_addr': '192.168.0.5'}, ...}}}
uplinkPortIds = []

qos/qos_sai_base.py:769: KeyError
`